include/SoapySDR/Convert.hpp

```diff
@@ -10,20 +10,36 @@
 
 #pragma once
 #include <SoapySDR/Config.h>
+#include <SoapySDR/Formats.hpp>
 #include <vector>
+#include <map>
 #include <string>
+#include <cstring> //memcpy
 #include <cstddef>
+#include <stdint.h>
+#include <stdexcept>
```

Comment:
```
+#include <SoapySDR/Formats.hpp>
 #include <vector>
 #include <vector>
+#include <map>
 #include <string>
 #include <string>
+#include <cstring> //memcpy
 #include <cstddef>
 #include <cstddef>
+#include <stdint.h>
+#include <stdexcept>
```

Keep the headers to a minimum. Like if they are only used inside the .cpp for implementation reasons, they should only be included there. Unless of course the a particular definition is actually used in the said header file